### ISSUE TITLE
[HC] [Enhancement]: "Interactive controls must not be nested" on product tabs (Accessibility enhancement)

### ISSUE THREAD

**Issue by OmarFPG** [woothemes/woocommerce#51333]
### Describe the solution you'd like

Ensure interactive controls are not nested, as they are not always announced by screen readers or can cause focus problems for assistive technologies.

Issue tested and confirmed on a clean install of WooCommerce, WordPress, and Storefront
![image](https://github.com/user-attachments/assets/01322fb6-2eb7-4043-9d76-799fcb614e57)

### Additional context

Reported here: https://wordpress.org/support/topic/accessibility-issue-29/
8719570-zen

**Comment by roykho** [comment]
Hi @OmarFPG - Thanks for reporting this issue. I've tried to reproduce this but everything seems to be working on my end. I am using the Voiceover utility on a Mac, when I tab to each of the "Description" and "Reviews" tabs, I am able to hear the correctly announcement.

Can you please give us reproducible steps?

**Comment by retrofox** [comment]
HI, @OmarFPG. Have you had the chance to look at Roy's comment? 🤞

**Comment by OmarFPG** [comment]
Hey team! @retrofox @roykho The test was made on Windows with the Axe extension for the Edge browser just as the merchant tried.

Considering a real life case use with a screen reader is working fine that'd have more precedence than a browser extension, but it's still worth checking just in case we're missing anything!

**Comment by itineris-milos** [comment]
Hi Guys,

I am not sure what is confusion here, we are keen to sort this as soon as possible.

Here is the live page where you can replicate

![Image](https://github.com/user-attachments/assets/48a2c983-5e05-4826-b7b5-0b0be7c863ea)

https://www.arhc.org.uk/product/christmas/christmas-cards/cambridge-market-square-at-christmas-time-pack-of-10-christmas-cards-with-envelopes/

More about it here: https://dequeuniversity.com/rules/axe/4.10/nested-interactive?application=AxeEdge

Let me know if you need more information

Thanks,
Milos

**Comment by itineris-milos** [comment]
Hi Guys,

Is there any update for the issue we have reported and it's been pending for a long time?

Thanks,
Milos

**Comment by gigitux** [comment]
Thanks for opening this issue! We’ve improved tab accessibility in [#53158](https://github.com/woocommerce/woocommerce/pull/53158). I re-ran the test and couldn’t reproduce the issue, so I’m closing it now.
